index.html

```diff
@@ -77,6 +77,7 @@
                 </div>
                 <div class="grid__unit">
                     <div style="min-height:50px;width:100%;padding:16px;background:#CCFFEC">
+                        <img src="unleash-logo.png" align="left" alt="Unleash logo" style="padding-right:30px">
```

Comment:
svg? point to some cdn-version instead of hard-coding here?

Reply:
Couldn't find an SVG version

Reply:
BTW, this is hosted at github.io, I'm pretty sure that's as good CDN as any

Reply:
It was more to avoid keeping a copy in this repo, and always get the latest. no issue, though

Reply:
Much better than nothing IMO!